DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan_suite/test_all_supported_modes.py

```diff
@@ -0,0 +1,186 @@
+import pytest
+
+from dent_os_testbed.lib.bridge.bridge_vlan import BridgeVlan
+from dent_os_testbed.utils.test_utils.tgen_utils import tgen_utils_get_dent_devices_with_tgen
+
+pytestmark = pytest.mark.suite_functional_vlan
+
+packet_vids = ['x', 1, 2, 3, 4]  # VLAN tag number contained in the transmitted packet
+vlans = {
+    2: {
+        "ports": [1, 2],
+        "pvid": False,
+        "untagged": False
+    },
+    3: {
+        "ports": [1],
+        "pvid": False,
+        "untagged": True
+    },
+    "3": {
+        "ports": [3],
+        "pvid": True,
+        "untagged": True
+    },
+    4: {
+        "ports": [1, 4],
+        "pvid": True,
+        "untagged": True
+    }
+}
+
+
+@pytest.mark.asyncio
+async def test_all_supported_modes_broadcast(testbed):
+    f"""
+    Test Name: Broadcast with all modes supported
+    Test Suite: suite_functional_vlan
+    Test Overview: Test VLAN with broadcast when all modes supported
+    Test Procedure:
+    1.Specify test data:
+        - packet_vids : {packet_vids}\n
+        - vlans\n {vlans}
+    2.Set links to vlans.
+        Verify vlans successfully added to links
+    3.Map receiving and non receiving ports.
+        Port 1 ->  tx_port
+        Ports 4 -> rx_ports
+    4.Setup packet stream(s) for the broadcast packet:
+        src_mac =
```

Comment:
Please specify MAC. If it doesn't matter what the MAC is then use:
```suggestion
        src_mac = 'XX:XX:XX:XX:XX:XX'
```